tests/unit/test_scenario.py

```diff
@@ -0,0 +1,59 @@
+from unittest.mock import ANY
+
+import ops
+import pytest
+from charm import HexanatorCharm
+from scenario import Container, Context, Relation, State
+
+
+def default_pebble_layer() -> dict:
+    import yaml
+
+    tmp = yaml.safe_load(open("rockcraft.yaml").read())
+    assert tmp["services"]["gubernator"]["startup"] == "enabled"
+    return tmp
+
+
+@pytest.fixture
+def ctx():
+    return Context(HexanatorCharm)
+
+
+@pytest.fixture
+def initial_state():
+    pebble_layers = {"default": ops.pebble.Layer(raw=default_pebble_layer())}  # type: ignore
```

Comment:
I think you could avoid the ignore by having the function return a LayerDict, not dict.